### PR TITLE
[feat] Create event on mobile app

### PR DESCRIPTION
**Pull request by rado196**
Create events functionality in mobile app.

This PR also includes:
1) Upgrading `React` and `React Native` to the latest versions (to fix build error: `ultras.app/Frameworks/RCTVibration.framework/Info.plist: no such file or directory`).
2) Upgrading packages to the latest versions.
3) Enabling `Hermes` JavaScript engine.
4) Typo fixes.

### CODE REVIEW

File: clients/app/src/views/components/base/FormControl/FormControl.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Box } from 'native-base';
+import { useTheme } from 'themes';
+import { IFormControlProps } from './types';
+
+const FormControl: React.FC<IFormControlProps> = ({ children }) => {
```
Comment (hay92k) [inline clients/app/src/views/components/base/FormControl/FormControl.tsx:6]: not the best name for this component

File: clients/app/src/views/components/base/Icon/Icon.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ const Icon: React.FC<IIconProps> = ({ name, color = 'iconSecondary', ...props })
 
   return (
     <NBIcon color={colors[color]} viewBox={icons[name].viewBox} {...props}>
-      <SvgXml xml={icons[name].content} />
+      <SvgXml xml={icons[name].content} fill={colors[color]} />
```
Comment (hay92k) [inline clients/app/src/views/components/base/Icon/Icon.tsx:13]: why we need this?
Isn't it bring bugs?

File: clients/app/src/views/components/base/RemoveButton/types.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export interface RemoveButtonInterface {
```
Comment (hay92k) [inline clients/app/src/views/components/base/RemoveButton/types.ts:1]: why nor IRemoveButtonProps

**Review by hay92k — CHANGES_REQUESTED** [review]

File: clients/app/src/views/components/base/Icon/Icon.tsx (hunk 2 above)
Comment (rado196) [reply clients/app/src/views/components/base/Icon/Icon.tsx:13]: that's required by hermes

File: clients/app/src/views/components/base/RemoveButton/types.ts (hunk 3 above)
Comment (rado196) [reply clients/app/src/views/components/base/RemoveButton/types.ts:1]: will change

File: clients/app/src/views/components/base/FormControl/FormControl.tsx (hunk 1 above)
Comment (rado196) [reply clients/app/src/views/components/base/FormControl/FormControl.tsx:6]: will update to `InputSection`

**Review by hay92k — APPROVED** [review]
lgtm

**Pull request merged**